syncstorage-spanner/src/pool.rs

```diff
@@ -84,6 +87,30 @@ impl SpannerDbPool {
             self.quota,
         ))
     }
+
+    /// Spawn a task to periodically evict idle connections. Calls wrapper sweeper fn
+    ///  to use pool.retain, retaining objects only if they are shorter in duration than
+    ///  defined max_idle.
+    pub fn spawn_sweeper(&self, interval: Duration) {
```

Comment:
If you do move this to the `impl DbPool` trait, you can keep the `fn sweeper()` as part of the general `impl` block.